src/managers/KeybindManager.cpp

```diff
@@ -144,11 +144,30 @@ CKeybindManager::CKeybindManager() {
         },
         nullptr);
 
+    m_pRepeatKeyTimer = makeShared<CEventLoopTimer>(
+        std::nullopt,
+        [this](SP<CEventLoopTimer> self, void* data) {
+            if (m_vActiveKeybinds.size() == 0 || g_pSeatManager->keyboard.expired())
+                return;
+
+            for (SKeybind* k : m_vActiveKeybinds) {
```

Comment:
what if two keybinds are separate?

consider one keybind for A, and one for B.

When pressing A, then B, both will start spamming at the same time. When they are spamming, pressing C will make it _instantly_ spam too.

Is this what we want?

Reply:
despite the name m_vActiveKeybinds is only used/added for repeat keybinds

additionally in the case where there 2 binds for different keys
I believe that would/should require using `bindte`
but even with that can't make it work (it prolly should, but maybe for another time)
(just the set of keybinds for 1 key gets ran)

likely releated to the vector being cleared in multiple places, for example `onKeyEvent()`
which should for now handle edge cases where multiple keybinds for different keys would get matched
since it seems to only run the last one

Reply:
mhm, okay